app/controllers/IndexController.scala

```diff
@@ -52,24 +54,32 @@ class IndexController @Inject()(
   }
 
   private def initialise(redirect: Result)(implicit request: IdentifierRequest[_]): Future[Result] = {
-    def identifier(enrolmentKey: String, identifierKey: String): Option[String] = request.user.enrolments.enrolments
+
+    def getIdentifierFromEnrolment(enrolmentKey: String, identifierKey: String): Option[String] = request.user.enrolments.enrolments
       .find(_.key equals enrolmentKey)
       .flatMap(_.identifiers.find(_.key equals identifierKey))
       .map(_.value)
 
-    val utr = identifier("HMRC-TERS-ORG", "SAUTR")
-    val urn = identifier("HMRC-TERSNT-ORG", "URN")
+    val utr = getIdentifierFromEnrolment("HMRC-TERS-ORG", "SAUTR")
+    val urn = getIdentifierFromEnrolment("HMRC-TERSNT-ORG", "URN")
+
+    val identifier = (utr, urn) match {
+      case (Some(_), None) => utr
+      case (None, Some(_)) => urn
+      case _ => None
+    }
```

Comment:
I believe here we would be able to do
```suggestion
    val utr = getIdentifierFromEnrolment("HMRC-TERS-ORG", "SAUTR")
    val urn = getIdentifierFromEnrolment("HMRC-TERSNT-ORG", "URN")

     val identifier = utr.getOrElse(urn).orElse(None)
```

I did this elsewhere for checking both enrolments